src/sections/Members-grid/index.js

```diff
@@ -36,9 +36,7 @@ const MembersGrid = props => {
                         <Row>
                             {props.data.allMdx.nodes.map(({ id, frontmatter, fields }) => (
                                 <Col xs={12} sm={6} md={6} lg={4} xl={3} key={id}>
-                                    <Link to={fields.slug}>
-                                        <ProfileCard frontmatter={frontmatter} />
-                                    </Link>
```

Comment:
Remove the `Link` tag since it is not used anywhere.

Reply:
I think, I have removed it already, and added a cardlink prop

Reply:
Yeah @dDab-panda, But, Did you removed it from the import?

Reply:
Oh forgot about that, will remove it